Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.Droid/Contoso.Forms.Puppet.Droid.csproj

```diff
@@ -31,7 +31,6 @@
     <WarningLevel>4</WarningLevel>
     <AndroidLinkMode>None</AndroidLinkMode>
     <EmbedAssembliesIntoApk>false</EmbedAssembliesIntoApk>
-    <AndroidEnableMultiDex>true</AndroidEnableMultiDex>
```

Comment:
are you sure about that? debug has always been so huge with Firebase

Reply:
It is actually still enabled, if you open the project in VS it will say multidex enabled. I think it's possible because it says multidex = true in the section above, which is for debug configurations of any platform